### PR TITLE
Added dialogbox when deleting & Swipe left to delete

### PR DESCRIPTION
**Pull request by SabaKhukhunashvili**
**_GCI 17_**

**Added:**  
            ✯ Showing dialog box when deleting contact.
            ✯ Added Swipe Left to delete

**Video Preview:** https://www.youtube.com/watch?v=2h0BKcKxhx8&feature=youtu.be

### CODE REVIEW

**Comment by zwliew** [comment]
Why not use the default dialog style?

**Comment by SabaKhukhunashvili** [comment]
i think it matches theme and looks nicer than default dialog layout :D

File: app/src/main/java/com/community/jboss/leadmanagement/DialogBox.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import com.community.jboss.leadmanagement.main.contacts.ContactsAdapter;
+
+
+public class DialogBox {
+    public void showAlert(Activity activity, final Contact contact, final ContactsAdapter adapter){
+        final Dialog dialog = new Dialog(activity);
+        dialog.requestWindowFeature(Window.FEATURE_NO_TITLE);
+        dialog.setContentView(R.layout.delete_dialog_box);
+
+        dialog.findViewById(R.id.dialog_yes).setOnClickListener(new View.OnClickListener() {
```
Comment (garg-anuj) [inline app/src/main/java/com/community/jboss/leadmanagement/DialogBox.java]: Use Butterknife for binding

File: app/build.gradle [hunk 2]
```diff
@@ -58,6 +58,9 @@ dependencies {
     //Sugar ORM
     implementation 'com.github.satyan:sugar:1.5'
 
+    // SwipeLeft
+    compile "com.daimajia.swipelayout:library:1.2.0@aar"
```
Comment (garg-anuj) [inline app/build.gradle]: Use implementation instead of compile

File: app/src/main/java/com/community/jboss/leadmanagement/DialogBox.java (hunk 1 above)
Comment (SabaKhukhunashvili) [reply app/src/main/java/com/community/jboss/leadmanagement/DialogBox.java]: Done :D

**Comment by m-murad** [comment]
Travis is failing. Fix it

**Pull request closed without merging**